vg/recorder/recorder_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package recorder
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/gonum/plot/vg"
+)
+
+func TestRecorder(t *testing.T) {
+	rec := New(72)
+	rec.Actions = append(rec.Actions, &FillString{Font: "Times-Roman", Size: 12, X: 0, Y: 10, String: "Text"})
+	rec.Comment("End of preamble")
+	rec.Scale(1, 2)
+	rec.Rotate(0.72)
+	rec.KeepCaller = true
+	rec.Stroke(vg.Path{{Type: vg.MoveComp, X: 3, Y: 4}})
+	for i, a := range rec.Actions {
+		if got := a.Call(); !strings.HasSuffix(got, want[i]) {
+			t.Errorf("unexpected action:\n\tgot: %#v\n\twant: %#v", got, want[i])
+		}
+	}
+
+	var replay Canvas
+	err := rec.ReplayOn(&replay)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	for i, a := range rec.Actions {
+		got := replay.Actions[i].Call()
+		want := a.Call()
+		if !strings.HasSuffix(want, got) {
+			t.Errorf("unexpected action:\n\tgot: %#v\n\twant: %#v", got, want)
+		}
+	}
+
+	replay.Reset()
+	rec.Actions = append(rec.Actions, &FillString{Font: "Foo", Size: 12, X: 0, Y: 10, String: "Bar"})
+	err = rec.ReplayOn(&replay)
+	if !strings.HasPrefix(err.Error(), "Unknown font: Foo.") {
+		t.Errorf("unexpected error: %v", err)
+	}
+}
+
+var want = []string{
```

Comment:
I don't think that this hits every method, does it?
It probably should just to verify that there aren't any big, obvious bugs in one of the Actions.